docs/node/migrations/overview.mdx

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+sidebar_label: "Overview"
+id: migrations-overview
+title: Migrations
+description: Overview of Kwil's migration system
+slug: /node/migrations/overview
+---
+
+This section provides an overview of the migration system in Kwil. Migrations are used to upgrade the network to a new version or to a new network with minimal downtime. Depending on the nature of the upgrade, migrations can be coordinated or uncoordinated.
```

Comment:
For the intro paragraph on this page, this is my rephrasing:

> This section provides an overview of the migration system in Kwil. When a software upgrade on an existing network cannot be accomplished with a coordination mechanism, a *network migration* is used to move to a new network deployed with the new software version. Kwil supports two types of migrations to perform this with minimal downtime.

On this Overview page, we're explaining why network migrations (creating a new network with data migrated from old network) may be necessary.  That reason being that a rolling upgrade won't work because consensus rules changed, and height-based coordination is difficult or impossible to do in all cases.  Thus, to make migrating state to a new network with a new genesis block and chain ID, we have two documented approaches.